### ISSUE TITLE
Top Filters

### ISSUE THREAD

**Issue by belg1o** [anosidium/HacKit-Feedback-And-Support#21]
Hey,

Could we get some filtering similar to HckrNews (https://hckrnews.com/) where it can pull all feeds but filter the top 10%, 20%, 50%, etc.

Alongside automatic refresh based on a specified interval, where'd it send a push notification that x new posts have been published

**Comment by anosidium** [comment]
Thanks for your suggestions.

I am working on another app so my time are invested in it. That website, [hckr news](https://hckrnews.com/) is a bit confusing. I clicked on the top 10, 20 and 50% and it is not clear how it determine the ranking. It seems to me that it is not based on the points count nor comments count.

As for the last suggestion, I am afraid that it is a lot of work to implement it.

**Comment by belg1o** [comment]
> Thanks for your suggestions.
> 
> I am working on another app so my time are invested in it. That website, [hckr news](https://hckrnews.com/) is a bit confusing. I clicked on the top 10, 20 and 50% and it is not clear how it determine the ranking. It seems to me that it is not based on the points count nor comments count.
> 
> As for the last suggestion, I am afraid that it is a lot of work to implement it.

Fair enough, do you think it'd be possible to add a top list based off scoring?

**Comment by anosidium** [comment]
Do you mean filter stories by points? A bit like how you can filter unread stories?

**Comment by belg1o** [comment]
> Do you mean filter stories by points? A bit like how you can filter unread stories?

Yes sir! If I could combine those two that'd be great.
